### PR TITLE
[FEATURE] Check authentication for Talawa-admin added

### LINKED ISSUE #715: [Feature Request]  Check authentication for Talawa-admin
**Is your feature request related to a problem? Please describe.**

Currently, in Talawa-admin, the routes are insecure and the authentication of the user is checked only at login. 
For more info PalisadoesFoundation/talawa-admin#341

**Describe the solution you'd like**

Query to check whether the user is authenticated or not is added.

### PR DESCRIPTION
**Pull request by asmitsirohi**
**What kind of change does this PR introduce?**

feature

Issue Number:

Fixes #715 

**Did you add tests for your changes?**

Yes

**Snapshots/Videos:**

* Query
Screenshot 2022-06-08 at 08-22-26 Playground - http __localhost 4000_graphql

* User is not authenticated
  Screenshot 2022-06-08 at 08-22-50 Playground - http __localhost 4000_graphql

* User is authenticated
Screenshot 2022-06-08 at 08-25-23 Playground - http __localhost 4000_graphql

**If relevant, did you update the documentation?**

No

**Summary**

This Pull request is the changes in the backend regarding PalisadoesFoundation/talawa-admin#340. 
In this, a query is added in order to check the authentication in talawa-admin.

**Does this PR introduce a breaking change?**

No

**Other information**

This pull request is under GSoC'22.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Review by JamaicanFriedChicken — CHANGES_REQUESTED** [review]

File: lib/resolvers/auth_query/checkAuth.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const userExists = require('../../helper_functions/userExists');
+
+module.exports = async (parent, args, context) => {
+  let userFound = await userExists(context.userId);
+  if (!userFound) throw new Error('User not found');
```
[the remark this answers is not in the document]
Comment (asmitsirohi) [reply lib/resolvers/auth_query/checkAuth.js]: Yes, there is a constant present in the constants file, I have fixed this.

File: tests/resolvers/auth_query/checkAuth.spec.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  let generatedEmail = `${shortid.generate().toLowerCase()}@test.com`;
+  userId = await getUserId(generatedEmail);
+});
+
+afterAll(() => {
+  database.disconnect();
+});
+
+describe('Testing check auth resolver', () => {
+  test('Testing is the user logged in or not', async () => {
```
Comment (JamaicanFriedChicken) [inline tests/resolvers/auth_query/checkAuth.spec.js:21]: misspelled `is` here, should be `if`

**Review by JamaicanFriedChicken — CHANGES_REQUESTED** [review]

**Review by yasharth291 — APPROVED** [review]

**Comment by palisadoes** [comment]
Please fix the failing tests.

**Comment by asmitsirohi** [comment]
@palisadoes. I have fixed the failed tests.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**